console/views/center_pool/groupapp_backup.py

```diff
@@ -74,6 +74,7 @@ def post(self, request, *args, **kwargs):
             # if service use custom service, can't backup
             use_custom_svc = groupapp_backup_service.check_backup_app_used_custom_volume(group_id)
             if use_custom_svc:
+                logger.info("use custom volume: {}".format(use_custom_svc))
```

Comment:
line 29, groupapp_backup_service.check_backup_condition need confirm that it is working.

Reply:
Don't understand what it means